dali/test/python/test_dali_cpu_only.py

```diff
@@ -805,6 +969,38 @@ def test_stack_cpu():
     for _ in range(3):
         pipe.run()
 
+def test_batch_permute_cpu():
+    pipe = Pipeline(batch_size=batch_size, num_threads=3, device_id=None)
+    data = fn.external_source(source = get_data, layout = "HWC")
+    perm = fn.batch_permutation(seed=420)
+    processed = fn.permute_batch(data, indices=perm)
+    pipe.set_outputs(processed)
+    pipe.build()
+    for _ in range(3):
+        pipe.run()
+
+def test_squeeze_cpu():
+    test_data_shape = [10, 20, 3, 1, 1]
```

Comment:
Why not using `check_single_input` here?

Reply:
Done